src/init.ts

```diff
@@ -45,13 +45,13 @@ export interface PackageJson {
   version?: string;
   devDependencies?: Bag<string>;
   scripts?: Bag<string>;
-  name: string;
-  description: string;
-  main: string;
-  types: string;
-  files: string[];
-  license: string;
-  keywords: string[];
+  name?: string;
```

Comment:
May be that I'm missing context - why are these all optional now?

Reply:
They should always have been optional. The schema for `package.json` isn't very strong and there is no requirement that all of these keys must be present.